crates/common/src/traits.rs

```diff
@@ -4,6 +4,35 @@ use alloy_json_abi::Function;
 use alloy_primitives::Bytes;
 use alloy_sol_types::SolError;
 use std::{fmt, path::Path};
+use tracing::warn;
+
+/// List of functions included with Foundry that are prefixed with 'assert' but are not intended to be used as assertions.
+static FOUNDRY_ASSERTION_FN_NAMES: &[&str] = &[
```

Comment:
This will be unintuitive for users, good eye catching this. We should change to `alert`.

Originally what I had in mind was `assertion`, but the assertions are quite different at the credible layer in term of structure, so I feel that a meaningful disambiguation is better here.